src/sqlfluff/rules/L012.py

```diff
@@ -1,28 +1,33 @@
 """Implementation of Rule L012."""
 
 from sqlfluff.rules.L011 import Rule_L011
+from sqlfluff.core.rules.doc_decorators import document_configuration
 
 
+@document_configuration
 class Rule_L012(Rule_L011):
     """Implicit/explicit aliasing of columns.
 
     Aliasing of columns to follow preference
-    (explicit using an `AS` clause is default).
+    (explicit using an ``AS`` clause is default).
 
-    NB: This rule inherits its functionality from :obj:`Rule_L011` but is
-    separate so that they can be enabled and disabled separately.
+    .. note::
+       This rule inherits its functionality from :obj:`Rule_L011` but is
+       separate so that they can be enabled and disabled separately.
```

Comment:
I wonder if we should remove this note. Do users care about this? They may not be Python developers, so the word "inherit" may not be familiar to them, also.

Reply:
Agreed. It's gone!